src/main/scala/org/allenai/dqa/labeling/DiagramFeatures.scala

```diff
@@ -0,0 +1,59 @@
+package org.allenai.dqa.labeling
+
+import scala.io.Source
+
+import edu.cmu.dynet._
+import edu.cmu.dynet.DyNetScalaHelpers._
+import edu.cmu.dynet.dynet_swig._
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+
+/**
+ * Features of points in a diagram. 
+ */
+case class DiagramFeatures(imageId: String, pointFeatures: Map[Point, FloatVector]) {
+
+  def getFeatures(part: Part) = {
```

Comment:
Minor: Add return type to function. The map lookup here will throw exception if a missing point is looked up, but I guess that's fine as it shouldn't happen.